### PR TITLE
Upgraded yargs package to fix #7444

### LINKED ISSUE #7444: Update yargs version - as current version is insecure
### Description
theia 0.16.1 core is using dependency called yargs ^11.1.0 that in turn uses yargs-parser 9.0.2 that was reported as vulnerable.
https://github.com/eclipse-theia/theia/blob/f4ee12a0e618cea3b11db741ef7740ed99fd74b6/packages/core/package.json#L48 

Scenario using the theia cli user can manipulate yargs parser to change runtime code.

### Reproduction Steps
vulnerability description:
https://snyk.io/vuln/SNYK-JS-YARGSPARSER-560381

**OS and Theia version:**
1.0.0 and master
**Diagnostics:** 
https://snyk.io/vuln/npm:yargs-parser

###Need to upgrade to higher version of yargs

### PR DESCRIPTION
**Pull request by c16a**
#### What it does
Upgades the version of yargs package to fix https://snyk.io/vuln/SNYK-JS-YARGSPARSER-560381

#### How to test
Tests can be run with `yarn test` from the `examples/browser` directory.

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -99,4 +99,4 @@
   "nyc": {
     "extends": "../../configs/nyc.json"
   }
-}
+}
```
Comment (vince-fugnitto) [inline packages/core/package.json:102]: @c16a can you revert this change, the newline should be present at the end of files.

Comment (c16a) [reply packages/core/package.json:102]: Oh sorry I was editing this in Gitpod and it looks like the new line was automatically removed. I'm trying to add one again, but it vanishes when I try to save it.

I'll try pushing from local.

**Comment by c16a** [comment]
@vince-fugnitto let me know if this is good to go.

**Comment by vince-fugnitto** [comment]
> @vince-fugnitto let me know if this is good to go.

A couple of things before proceeding:
- Please be sure to sign the [Eclipse Contributor Agreement (ECA)](https://github.com/eclipse-theia/theia/blob/master/CONTRIBUTING.md#eclipse-contributor-agreement) so we can accept your changes. The last time I verified your sign-off email, no ECA was on file:

<div align='center'>

Screen Shot 2020-04-01 at 9 25 32 AM

</div>

- Please be sure to squash all commits. In general, we ask that a single commit be present in a pull-request unless commits can be logically separated.

**Comment by c16a** [comment]
@vince-fugnitto - done with both. Can you check if my ECA shows up now?

**Comment by vince-fugnitto** [comment]
> @vince-fugnitto - done with both. Can you check if my ECA shows up now?

Yes, the ECA looks good now thank you!
I'll wait for Travis to be green before merging, thank you for your first contribution 👍 

<div align='center'>

Screen Shot 2020-04-01 at 9 54 27 AM


</div>

**Comment by c16a** [comment]
@vince-fugnitto - thanks! Looks everything is good now.

**Comment by vince-fugnitto** [comment]
> @vince-fugnitto - thanks! Looks everything is good now.

Was there a reason for not updating other extensions?
We also have `yargs` in these places:

https://github.com/eclipse-theia/theia/blob/f4ee12a0e618cea3b11db741ef7740ed99fd74b6/dev-packages/cli/package.json#L48

https://github.com/eclipse-theia/theia/blob/f4ee12a0e618cea3b11db741ef7740ed99fd74b6/dev-packages/electron/package.json#L30

`cli` will be in fact a `devDependency` in the end, but not `electron` as it is mandatory for electron-based theia applications. In order to align, I think we should update both of them similarly to `core.

**Comment by marechal-p** [comment]
Note: Did not verify that it wasn't updated in the other places, but given what yarn resolved with your first modification, just update the other places and it should be good to go.

**Comment by akosyakov** [comment]
@marechal-p Anything else should be done to merge it?

**Comment by vince-fugnitto** [comment]
> @marechal-p Anything else should be done to merge it?

We should update the other [occurrences](https://github.com/eclipse-theia/theia/pull/7476#issuecomment-607279958) of `yargs` in other extensions, we can do so by updating the pull-request ourselves as there hasn't been much activity lately.

**Comment by akosyakov** [comment]
@vince-fugnitto fine with me
